### PR TITLE
feat(application-system): pay application fullnadarskirteini

### PR DESCRIPTION
**Pull request by jGeir**
## What

Fixing the final steps of driving license - avoiding infinite callback loop and fleshing out the pages later on. Now it also blocks user from finishing the application if the application payment is unconfirmed.

## Why

There were some bugs and unfinished work that was left behind due to other pressing matters.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request
Screenshot from 2021-07-08 10-09-55
Screenshot from 2021-07-08 13-03-56
Screenshot from 2021-07-08 12-48-22

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/driving-license/src/fields/Congratulations.tsx [hunk 1]
```diff
@@ -15,42 +13,40 @@ interface PropTypes extends FieldBaseProps {
...
-          </Accordion>
+        <Box marginTop={6}>
+          <Text>
+            Ef beðið var um viðbótargögn (nýja ljósmynd eða læknisvottorð) þarf
+            að skila þeim til Sýslumanns svo að fullnaðarskírteini fari í
+            pöntun.
+          </Text>
+          <Text marginTop={4}>
+            Ef svo var ekki þá verður fullnaðarskírteinið tilbúið á
+            afhendingarstað eftir 3 til 4 vikur.
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/fields/Congratulations.tsx:45]: wondering if it is probably better to keep the text strings inside the messages.ts? :)

File: libs/application/templates/driving-license/src/fields/Congratulations.tsx [hunk 2]
```diff
@@ -15,42 +13,40 @@ interface PropTypes extends FieldBaseProps {
...
           <AlertMessage
             type="success"
-            title="Til hamingju"
-            message={`
-              fullnaðarskírteinið þitt hefur verið samþykkt og mun vera tilbúið
-              hjá Sýslumanni í Reykjavík eftir 4 daga. Þegar þú sækir nýtt skírteini
-              þarftu að skila inn bráðabirgðaskírteininu þínu.
-            `}
+            title={`Til hamingju ${name.fullName}`}
+            message={`Umsókn þín um fullnaðarskírteini tókst.`}
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/fields/Congratulations.tsx:33]: noticed that you've added this to messages.ts but perhaps forgetting to use here?

File: libs/application/templates/driving-license/src/fields/ExamplePaymentPendingField.tsx [hunk 3]
```diff
@@ -46,11 +45,33 @@ export const ExamplePaymentPendingField: FC<Props> = ({
       {error && { error }}
 
       {!paymentStatus.fulfilled && (
-        <Box>
-          <Text variant="h2">{m.examplePaymentPendingField}</Text>
-          <Text marginTop="gutter">{m.examplePaymentPendingDescription}</Text>
+        <Box height="full">
+          <Text variant="h3">Augnablik meðan beðið er eftir staðfestingu</Text>
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/fields/ExamplePaymentPendingField.tsx:49]: same as above - 'noticed that you've added this to messages.ts but perhaps forgetting to use here?'

Comment (jGeir) [reply libs/application/templates/driving-license/src/fields/ExamplePaymentPendingField.tsx]: Fixed, Thanks.

**Review by albinagu — APPROVED** [review]
lgtm

**Review by eirikurn — APPROVED** [review]

**Pull request merged**